packages/shared/src/utils/url/url.ts

```diff
@@ -35,6 +40,23 @@ export const urlForLanguage = (lang: string, url: string = window.location.href)
     return `${current_url}`;
 };
 
+export const urlSetQuery = (queryObj: TQueryObj, url: string = window.location.href) => {
+    const current_url = new URL(url);
+
+    // eslint-disable-next-line no-restricted-syntax
+    for (const [key, value] of Object.entries(queryObj)) {
+        if (value) {
+            current_url.searchParams.set(key, value);
+        } else {
+            current_url.searchParams.delete(key);
+        }
+    }
+
+    current_url.searchParams.sort();
```

Comment:
@sergei-deriv Do we need to sort? I don’t think that makes any difference 🤔

Reply:
@farzin-deriv it makes difference, because live-chat account has URL rules and if we write rule '?dark=1&lang=ES' it's not the same for live-chat as '?lang=ES&dark=1'. So we need a stable order of parameters.

Reply:
@sergei-deriv Thank you for your explanation 🙇🏻